bait-files/src/main/java/com/google/LoginFilter.java

```diff
@@ -58,31 +53,30 @@ public void doFilter(ServletRequest req, ServletResponse res, FilterChain chain)
       }
     }
     
-    // Case: User is logged in
-    else {
+    // Case: User is logged in but not registered (will replace else if with code checking registered)
+    else if (1!=1){
         /* TODO: Add case when user is not registered */
     }
 
-    // Is logged in and registered but is trying to access profile jsp or now restricted register
-    // page
-    if (request.getRequestURI().endsWith("profile.jsp")
-        || request.getRequestURI().endsWith("register.jsp")
-        || request.getRequestURI().endsWith("login.jsp")) {
-      RequestDispatcher requestDispatcher = request.getRequestDispatcher("WEB-INF/profile.jsp");
-      requestDispatcher.forward(request, response);
-      return;
+    else {
+    
+        // Is logged in and registered but is trying to access profile jsp or now restricted register
+        // page
+        if (request.getRequestURI().endsWith("profile.jsp")
+            || request.getRequestURI().endsWith("register.jsp")
+            || request.getRequestURI().endsWith("login.jsp")) {
+            RequestDispatcher requestDispatcher = request.getRequestDispatcher("WEB-INF/login.jsp");
+            requestDispatcher.forward(request, response);
+            return;
+    } else {
```

Comment:
You have two else statements in a row here.. making this second one impossible to hit.  So I would delete it and then put a TODO comment after the existing if-else structure saying to handle the case where the user is logged in and registered and wants to access site resources.

Reply:
Added indentation to resolve confusion